assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js

```diff
@@ -69,6 +69,8 @@
                         vm.repoChecks.valid = Object.keys(vm.repoChecks.checks).every(function(k) {
                             return vm.repoChecks.checks[k].status === true
                         });
+                        // Enable next button if all the checks pass
+                        $scope.upgradeVm.steps.activeStep.finished = vm.repoChecks.valid;
```

Comment:
Child controllers shouldn't have visibility over its parents. We need to find an alternative way to expose the completion of the step.